src/components/editor/EditorLogin.jsx

```diff
@@ -51,6 +85,29 @@ export default class Login extends BaseComponent {
     }
   }
 
+  onGoogleResponse(response) {
+    if (response.error) {
+      alert(`Google Login Failed.\n${response.error}`);
+      return;
+    }
+
+    const token = response.tokenId;
+
+    const xhr = new XMLHttpRequest();
+    xhr.open('POST', '/googlelogin', false);
+    xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+
+    xhr.onreadystatechange = () => {
+      if (xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
+        browserHistory.push('/');
```

Comment:
Seem line 75-84 in this same file on how the current login API is implemented and copy that here

Reply:
okay